script/Deployer.s.sol

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {Script} from "../lib/forge-std/src/Script.sol";
+
+import { stdJson } from "../lib/forge-std/src/StdJson.sol";
+import "../contracts/ResultManager.sol";
+import "../contracts/Forwarder.sol";
+import "../contracts/TransparentForwarder.sol";
+import "../contracts/Staking.sol";
+
+contract Deployer is Script {
+        using stdJson for string;
+        address public SIGNER_ADDRESS = vm.envAddress("SIGNER_ADDRESS");
+        bytes4 public RESULTGETTER_SELECTOR = 0xadd4c784;  
+        bytes4 public UPDATE_SELECTOR = 0x2d444fd5; 
+        bytes4 public VALIDATE_SELECTOR = 0x41417a9d;
+        address public deployer;
+        string internal _deployments;
+        string internal _deploymentsPath;
+
+    function run() public {
+
+        deployer = vm.envAddress("DEPLOYER_ADDRESS");
+        string memory network = vm.envString("NETWORK");
+        _deploymentsPath = string.concat(string.concat("./deployments/", network), ".json");
+
+         vm.startBroadcast(deployer);
+
+        // Deploy ResultManager
```

Comment:
the validateTimestamp check needs to be added. If it can't be added then create a shell script that first runs the validate timestamp check and once passed, then deploy the contracts